### ISSUE TITLE
Unclear what was mounted

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/libelektra#1290]
As discussed with @krit0n 

`kdb list-plugins <key name>`

returns the list of plugins that would be used for key name.

- [ ] `kdb mount -v`

list plugins per mountpoint

- [ ] `kdb gmount -v`

list plugins for global mount

- [x] `kdb global-umount` is missing, see #1132

**Comment by sanssecours** [comment]
I really like the idea of listing the plugins for a specific mountpoint via `kdb mount -v` (or even just `kdb mount`). It would be really awesome if someone implements this functionality 🙏.

**Comment by markus2330** [comment]
`kdb mountpoint-info <mountpoint>` lists plugins, for example:

```
> kdb mountpoint-info /hosts                                                                                                                                                                                                     Dienstag 23:24:41
Version: 0.8.19
Default resolver: resolver_fm_hpu_b
Default storage: dump
Mountpoint: /hosts
File: hosts
        config:
        glob/set/#0 = /ipv4/*
        glob/set/#1 = /ipv6/*
        glob/set/#2 = /ipv4/*/*
        glob/set/#3 = /ipv6/*/*
        glob/set/#4/ = /*
        path = hosts
getplugins:
        #0#resolver
        #5#hosts#hosts#
        #6#glob#glob#
setplugins:
        #0#resolver
        #1#glob
        #2#error#error#
        #3#network#network#
        #5#hosts
        #6#sync#sync#
        #7#resolver
errorplugins:
        #5#resolver_fm_hpu_b#resolver#
```

Merged with #1364. The output may be improved by removing the `#`, though.
